datasets/interpress_news_category_tr_lite/README.md

```diff
@@ -0,0 +1,150 @@
+---
+annotations_creators:
+- found
+language_creators:
+- found
+languages:
+- tr
+licenses:
+- unknown
+multilinguality:
+- monolingual
+size_categories:
+- 100k<n<1M
+source_datasets:
+- original
```

Comment:
```suggestion
- extended|interpress_news_category_tr
```
This dataset is based on the interpress_news_category_tr one right ?

Reply:
True